frontend/src/containers/RouterManager/index.tsx

```diff
@@ -1,10 +1,15 @@
+import { RequireAuth } from 'hoc';
 import { FC } from 'react';
 import { Route, Routes } from 'react-router-dom';
 import { routes } from 'router';
 
 const RouteManager: FC = () => {
   const router = routes.map((route) => {
-    return <Route path={route.path} element={route.component} key={route.name} />;
+    return <Route path={route.path} element={
```

Comment:
Read about React laze and try to add component, witch will create lazy modules, if it module(page) is protectd - redirect to home page for example